integrated_channels/canvas/client.py

```diff
@@ -559,7 +598,18 @@ def _delete(self, url):
         Args:
             url (str): The canvas url to send delete requests to.
         """
+        start_time = time.time()
         delete_response = self.session.delete(url, data='{"event":"conclude"}')
+        duration_seconds = time.time() - start_time
+        stringify_and_store_api_record(
+            enterprise_customer=self.enterprise_configuration.enterprise_customer,
+            enterprise_customer_configuration_id=self.enterprise_configuration.id,
+            endpoint=url,
+            data='{"event":"conclude"}',
```

Comment:
we can create a variable for it and use on line 602 as well

Reply:
yes a variable can be used
updated.